packages/upstream/tar.0.9.0/opam

```diff
@@ -14,11 +14,10 @@ build: [
 
 depends: [
   "jbuilder"          {build & >="1.0+beta7"}
-  "ppx_tools"         {build}
-  "ppx_cstruct"       {build}
+  "ppx_tools"
+  "ppx_cstruct"
   "cstruct"           {>= "1.9.0"}
   "re"
   "result"
 ]
 available: [ ocaml-version >= "4.03.0" ]
-patches: [ "0001-Do-not-always-build-with-safe-string-this-can-break-.patch" ]
```

Comment:
You may want to delete this file if we are not using it. Would probably be useful to do a jenkins build, but I think that aside from oxenstored we are safe-string compatible now so it should be fine.

Reply:
So let's keep this until we switch the compiler. I'll put it back in.